src/main/java/com/tabnine/inline/CompletionPreview.java

```diff
@@ -97,6 +97,18 @@ public void togglePreview(CompletionOrder order) {
     completionsEventSender.sendToggleInlineSuggestionEvent(order, currentIndex);
   }
 
+  @Nullable
+  public static TabNineCompletion getCurrentCompletion(Editor editor) {
```

Comment:
I think it will be better to move this function up to be near the other static methods